### ISSUE TITLE
Send string in POST request

### ISSUE THREAD

**Issue by cfoster5** [capacitor-community/http#43]
I am trying to make a POST request to an endpoint with the following:

```
async function getGames() {
    const response = await Http.request({
      method: 'POST',
      url: 'https://api-v3.igdb.com/games',
      headers: {
        'user-key': "xxxxxx",
        'Accept': "application/json"
      },
      data: "fields *, cover.*, release_dates.*; limit 50;"
    });
    return response;
  }
```

I am not able to send a string with the data property in this request.

**Comment by Sheffer** [comment]
As per the example, I think that data is supposed to be an object, such as
```
data: {
      foo: 'bar',
      cool: true
    }
```

not a string.

**Comment by cfoster5** [comment]
I believe you are correct, however, this seems like a big limitation.

**Comment by thomasvidas** [comment]
Fixed in #141
